### PR TITLE
Allow easy Docker port override

### PR DESCRIPTION
**Pull request by Frenzie**
Because I use 8080 for other things. ;-)

### CODE REVIEW

**Review by Alkarex — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -4,6 +4,8 @@ ifndef TAG
 	TAG=dev-alpine
 endif
 
+PORT ?= 8080
```
Comment (aledeg) [inline Makefile:7]: I would use a less generic envvar to avoid collision. Maybe something like ```FRSS_PORT```.

**Review by aledeg — DISMISSED** [review]

File: Makefile (hunk 1 above)
Comment (marienfressinaud) [reply Makefile:7]: The command is usually used as `PORT=8000 make start` (i.e. no need to export) so I don't mind having a generic variable name. If we change to `FRSS_PORT`, we should change `TAG` as well to be coherent

Comment (aledeg) [reply Makefile:7]: That makes sense

**Review by marienfressinaud — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (Frenzie) [reply Makefile:7]: Or `make start PORT=23423` (doesn't change the point). :-)

**Pull request merged**